plugins/kernelbook/skills/cuda-matmul-tmem-load/SKILL.md

```
@@ -0,0 +1,249 @@
+
+| Instruction | Direction | Description |
+|-------------|-----------|-------------|
+| `tcgen05.ld` | TMEM → Registers | Load accumulator for epilogue |
+| `tcgen05.st` | Registers → TMEM | Store data to TMEM |
+| `tcgen05.cp` | SMEM → TMEM | Copy from shared memory |
+
+## TMEM Load (tcgen05.ld)
+
+### Instruction Format
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)